gradle/libs.versions.toml

```diff
@@ -23,6 +27,10 @@ coroutines = "1.7.3"
 
 [libraries]
 android-buildTools = { module = "com.android.tools.build:gradle", version.ref = "agp" }
+androidx-lifecycle-runtime-ktx = { module = "androidx.lifecycle:lifecycle-runtime-ktx", version.ref = "lifecycleViewmodelKtx" }
+androidx-lifecycle-viewmodel-ktx = { module = "androidx.lifecycle:lifecycle-viewmodel-ktx", version.ref = "lifecycleViewmodelKtx" }
+androidx-lifecycle-viewmodel-ktx-v230 = { module = "androidx.lifecycle:lifecycle-viewmodel-ktx", version.ref = "androidx-lifecycle-lifecycleViewmodelKtx" }
+androidx-lifecycle-viewmodel-ktx-v240 = { module = "androidx.lifecycle:lifecycle-viewmodel-ktx", version.ref = "androidx-lifecycleViewmodelKtx" }
```

Comment:
Why do we need 3 different versions of the same lib?

Reply:
@theolm I tried 3 different mvvm libs but none of them were recognized by gradle. I don't know why they weren't removed from here. Please, let's cover this topic Today.

Reply:
Done